### DISCUSSION TITLE
Issue with capacitor Ionic/Vue

### DISCUSSION THREAD

**Discussion by creativalab** [NiklasMerz/cordova-plugin-fingerprint-aio#349] [Q&A] [answered]
Hi, i need to kwno how to use biometric params with Iionic/Vue/Capacitor.
In 5.0.0-beta.21 is not possibile to use biometric param.
In v4 missing ngx folder, so i can't use it.

Thanks for your help

**Comment by creativalab** [comment]
With v4 i have this issue Uncaught TypeError: Object(...) is not a function refers to Cordova() in index.js line 92

**Comment by NiklasMerz** [comment, marked as answer]
If you have issues with ionic-native please check their issues on Github.
